kahuna/public/js/components/gr-collection-overlay/gr-collection-overlay.css

```diff
@@ -40,6 +40,13 @@
     width: 365px;
     background: #444;
     z-index: 10;
+    display: flex;
+    justify-content: space-between;
+    align-items: center;
+}
+
+.collection-overlay__collections__heading-text {
+  margin: 0 10px 0 0;
```

Comment:
super minor: `margin-left` to be slightly less verbose

Reply:
It's a `<p>` tag which comes with default browser styles since we're not using a rest 😢